netlify.toml

```diff
@@ -4,24 +4,24 @@
 # DO NOT REMOVE THIS (contact @kubernetes/sig-docs-leads)
 publish = "public"
 functions = "functions"
-command = "make non-production-build"
+command = "cd themes/docsy && git submodule update -f --init && cd ../.. && make non-production-build"
 
 [build.environment]
-HUGO_VERSION = "0.59.1"
+HUGO_VERSION = "0.59.1 "
```

Comment:
remove space
Any chance you have tested with Hugo v0.70.0?

Reply:
Afraid not - Docsy only required ~0.53 extended